org.flathub.flatpak-external-data-checker.yml

```diff
@@ -0,0 +1,143 @@
+id: org.flathub.flatpak-external-data-checker
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+finish-args:
+  - --share=network
+  - --filesystem=host:ro
+command: flatpak-external-data-checker
+cleanup:
+  - "*.a"
+  - "*.la"
+  - /include
+  - /lib/pkgconfig
+  - /share/man
+  - /share/doc
+modules:
+
+  - name: flatpak-external-data-checker
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib /app/bin
+      - cp -ax . /app/lib/fedc
+      - ln -sr /app/{lib/fedc,bin}/flatpak-external-data-checker
+      - ln -sr /app/{lib/fedc,bin}/canonicalize-manifest
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: git
+        url: "https://github.com/flathub/flatpak-external-data-checker.git"
```

Comment:
Use the latest available commit here for now.

Reply:
Done.